packages/core/test/combinator/mergeConcurrently-test.js

```diff
@@ -21,13 +20,11 @@ describe('mergeConcurrently', () => {
     const n = 3
 
     return collectEvents(take(n, s), ticks(n))
-      .then(events => {
-        assert.equals(events, [
+      .then(eq([
```

Comment:
Love the use of eq() here.  However, I think the objects below are indented an extra level.  I wonder why eslint isn't complaining ... hmmm.